blockscout-ens/bens-server/config/prod.json

```diff
@@ -535,7 +543,26 @@
           "icon_url": "https://i.imgur.com/yYc2C6V.png",
           "docs_url": "https://docs.space.id/"
         }
+      },
+      "gravity": {
+        "disabled": true,
+        "tld_list": ["g"],
+        "network_id": 1624,
+        "subgraph_name": "gravity-subgraph",
+        "address_resolve_technique": "reverse_registry",
+        "specific": {
+          "type": "ens_like",
+          "native_token_contract": "0x0000000000000000000000000000000000000000",
+          "empty_label_hash": "0x702d9dc001440f23c84ab0d0d63e1dd6b0d62b077c2d9a4694ef997a19ddd38a"
```

Comment:
I just check and correct values is `0x438c16e05212074ed5b2d4beba06988729060b93431f68bf346c93c64a1ce640`